tests/github-action-helper.sh

```diff
@@ -32,6 +32,7 @@ deploy_rook() {
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/operator.yaml
   curl https://raw.githubusercontent.com/rook/rook/master/deploy/examples/cluster-test.yaml -o cluster-test.yaml
   sed -i "s|#deviceFilter:|deviceFilter: $(lsblk | awk '/14G/ {print $1}' | head -1)|g" cluster-test.yaml
+  sed -i '34s/1/3/' cluster-test.yaml
```

Comment:
The line number could change in the yaml. Instead could you replace the first instance of `count: 1`?